src/v2/view-builder/components/Link.js

```diff
@@ -19,6 +19,23 @@ const Link = View.extend({
 
   tagName: 'a',
 
+  /**
+   * View
+   * adds any view under the link
+   */
+  linkInfo: null,
+
+  initialize() {
+    View.prototype.initialize.apply(this, arguments);
+    if (this.options.linkInfo) {
```

Comment:
adding the view in link, means we inherit the styling

<img width="405" alt="Screen Shot 2021-06-28 at 3 31 05 PM" src="https://user-images.githubusercontent.com/19613909/123711897-15bd0c80-d826-11eb-89e6-62fcf3d0d266.png">